src/component/strategies/StrategyForm/StrategyParameters/StrategyParameters.tsx

```diff
@@ -0,0 +1,37 @@
+import { StrategyParameter } from './StrategyParameter/StrategyParameter';
+import React from 'react';
+import { ICustomStrategyParameter } from 'interfaces/strategy';
+
+function gerArrayWithEntries(num: number) {
+    return Array.from(Array(num));
+}
+
+interface IStrategyParametersProps {
+    input: ICustomStrategyParameter[];
+    count: number;
+    updateParameter: (index: number, updated: object) => void;
+    setParams: React.Dispatch<React.SetStateAction<ICustomStrategyParameter[]>>;
+    errors: { [key: string]: string };
+}
+
+export const StrategyParameters = ({
+    input = [],
+    count = 0,
+    updateParameter,
+    setParams,
+    errors,
+}: IStrategyParametersProps) => (
+    <div>
+        {gerArrayWithEntries(count).map((v, i) => (
```

Comment:
Maybe we can use `input.map((item, i) => ...` insteead of the count prop.

Reply:
Let's also use proper parameter names here `value / index`